src/main/resources/db/migration/postgres/V2.4.1.0__data_column_order_states.sql

```diff
@@ -0,0 +1,338 @@
+									  version, experiment_state_id)
+			select nextval('value_pkseq'), 'data column', 'ACAS DDICT', 'column name', 'data column_column name', column_name, false, false,
+									  'column name', transaction_id, 'codeValue', 'codeValue_column name', true, 'ACAS', now(),
+									  0, expt_state_id;
+		-- create units value
+		insert into experiment_value (id, code_kind, code_origin, code_type, code_type_and_kind, code_value, deleted, ignored, 
+									  ls_kind, ls_transaction, ls_type, ls_type_and_kind, public_data, recorded_by, recorded_date, 
+									  version, experiment_state_id)
+			select nextval('value_pkseq'), 'data column', 'ACAS DDICT', 'column units', 'data column_column units', units, false, false,
+									  'column units', transaction_id, 'codeValue', 'codeValue_column units', true, 'ACAS', now(),
+									  0, expt_state_id;
+		-- create concentration value
+		insert into experiment_value (id, deleted, ignored, numeric_value,
+									  ls_kind, ls_transaction, ls_type, ls_type_and_kind, public_data, recorded_by, recorded_date, 
+									  version, experiment_state_id)
+			select nextval('value_pkseq'), false, false, concentration,
+									  'column concentration', transaction_id, 'numericValue', 'numericValue_column concentration', true, 'ACAS', now(),
+									  0, expt_state_id;
+		-- create conc_units value
+		insert into experiment_value (id, code_kind, code_origin, code_type, code_type_and_kind, code_value, deleted, ignored, 
+									  ls_kind, ls_transaction, ls_type, ls_type_and_kind, public_data, recorded_by, recorded_date, 
+									  version, experiment_state_id)
+			select nextval('value_pkseq'), 'data column', 'ACAS DDICT', 'column conc units', 'data column_column conc units', conc_units, false, false,
+									  'column conc units', transaction_id, 'codeValue', 'codeValue_column conc units', true, 'ACAS', now(),
+									  0, expt_state_id;
+		-- create hidden value
+		insert into experiment_value (id, code_kind, code_origin, code_type, code_type_and_kind, code_value, deleted, ignored, 
+									  ls_kind, ls_transaction, ls_type, ls_type_and_kind, public_data, recorded_by, recorded_date, 
+									  version, experiment_state_id)
+			select nextval('value_pkseq'), 'boolean', 'ACAS DDICT', 'boolean', 'boolean_boolean', case when hidden then 'TRUE' else 'FALSE' end, false, false,
+									  'hide column', transaction_id, 'codeValue', 'codeValue_hide column', true, 'ACAS', now(),
+									  0, expt_state_id;
+		-- create condition value
+		insert into experiment_value (id, code_kind, code_origin, code_type, code_type_and_kind, code_value, deleted, ignored, 
+									  ls_kind, ls_transaction, ls_type, ls_type_and_kind, public_data, recorded_by, recorded_date, 
+									  version, experiment_state_id)
+			select nextval('value_pkseq'), 'boolean', 'ACAS DDICT', 'boolean', 'boolean_boolean', case when condition then 'TRUE' else 'FALSE' end, false, false,
```

Comment:
This would obviously have ramifications for the ACAS and R code.  I'd have to check but I believe lines like this get json converted into lowercase `true` values.

https://github.com/mcneilco/acas/blob/38675b89ff98d7a004a8f515bd5c4bf9b56c9fcf/modules/GenericDataParser/src/server/generic_data_parser.R#L2043

Reply:
@brianbolt I agree with you in general that we have a preference for lowecase true/false
However in looking at the historically loaded "data column order" experiment values (created by SEL), the "hide" and "condition" `stringValue`s are `TRUE` / `FALSE`.

I tried to match that pattern here, to keep the new data I insert consistent with preexisting data.

I noticed the same issue you've pointed out in ACASFormFields, and made this fix: https://github.com/mcneilco/acas/pull/1033/files#diff-21cee30a2860a441853c050a1d31946bdcf5945f65d07bcc5a1f5dbad1cf0420R1037